UnityProjects/MRTKDevTemplate/Assets/XR/Settings/OpenXR Package Settings.asset

```diff
@@ -108,6 +108,7 @@ MonoBehaviour:
   priority: 0
   required: 1
   disableFirstPersonObserver: 0
+  enablePoseUpdateOnBeforeRender: 0
```

Comment:
Why are these set to zero? Sounds useful...

Reply:
Because perf 😬 it's part of some hands perf work we've been doing. by default, most people aren't using these values, so we're turning them off. but! if you need them, you can check the box and get them. they're also always query-able via APIs like `HandTracker`